firefox-ios/Client/Frontend/Home/Homepage Rebuild/TopSites/TopSiteState.swift

```diff
@@ -75,11 +73,21 @@ final class TopSiteState: Hashable, Equatable {
 
     // MARK: - Equatable
     static func == (lhs: TopSiteState, rhs: TopSiteState) -> Bool {
-        lhs.site == rhs.site
```

Comment:
needed to add these, because `site` was not enough to differentiate the item's state, so pinning + unpinning didn't update the UI appropriately.